yt_dlp/extractor/afreecatv.py

```diff
@@ -61,46 +61,46 @@ def _perform_login(self, username, password):
 
     def _call_api(self, endpoint, display_id, data=None, headers=None, query=None):
         return self._download_json(Request(
-            f'https://api.m.afreecatv.com/{endpoint}',
+            f'https://api.m.sooplive.co.kr/{endpoint}',
             data=data, headers=headers, query=query,
             extensions={'legacy_ssl': True}), display_id,
             'Downloading API JSON', 'Unable to download API JSON')
 
 
 class AfreecaTVIE(AfreecaTVBaseIE):
-    IE_NAME = 'afreecatv'
-    IE_DESC = 'afreecatv.com'
+    IE_NAME = 'soop'
+    IE_DESC = 'sooplive.co.kr'
     _VALID_URL = r'''(?x)
                     https?://
                         (?:
-                            (?:(?:live|afbbs|www)\.)?afreeca(?:tv)?\.com(?::\d+)?
+                            (?:(?:live|afbbs|www)\.)?sooplive\.co\.kr(?::\d+)?
                             (?:
                                 /app/(?:index|read_ucc_bbs)\.cgi|
                                 /player/[Pp]layer\.(?:swf|html)
                             )\?.*?\bnTitleNo=|
-                            vod\.afreecatv\.com/(PLAYER/STATION|player)/
+                            vod\.sooplive\.co\.kr/(PLAYER/STATION|player)/
                         )
                         (?P<id>\d+)/?(?:$|[?#&])
                     '''
     _TESTS = [{
-        'url': 'http://live.afreecatv.com:8079/app/index.cgi?szType=read_ucc_bbs&szBjId=dailyapril&nStationNo=16711924&nBbsNo=18605867&nTitleNo=36164052&szSkin=',
+        'url': 'http://live.sooplive.co.kr:8079/app/index.cgi?szType=read_ucc_bbs&szBjId=dailyapril&nStationNo=16711924&nBbsNo=18605867&nTitleNo=36164052&szSkin=',
```

Comment:
Revert; changing dead URLs to non-existent URLs doesn't make sense. If this type of URL doesn't exist anymore then we should just remove these tests entirely

```suggestion
        'url': 'http://live.afreecatv.com:8079/app/index.cgi?szType=read_ucc_bbs&szBjId=dailyapril&nStationNo=16711924&nBbsNo=18605867&nTitleNo=36164052&szSkin=',
```